Library/Homebrew/utils.rb

```diff
@@ -376,6 +376,14 @@ def ignore_interrupts(opt = nil)
   trap("INT", std_trap)
 end
 
+def capture_stderr
```

Comment:
Yeh, consolidation there makes sense although might want to tweak the name if we do that too. Definitely one for another PR 👍